packages/hash/frontend/src/blocks/page/PageBlock.tsx

```diff
@@ -83,7 +83,7 @@ export const PageBlock: FunctionComponent<PageBlockProps> = ({
   return (
     <UserBlocksProvider value={blocks}>
       <BlockLoadedProvider routeHash={routeHash}>
-        <div id="root" ref={root} />
+        <div id="root" ref={root} style={{ position: "relative" }} />
```

Comment:
Nit: Let's use MuiBox here and add the style via the sx prop

Reply:
Done in 30521136de117ad64c8a37ddeca6f71bcab0d874